app/src/androidTest/java/org/simple/clinic/benchmark/CanaryBenchmarkTest.kt

```diff
@@ -1,11 +1,33 @@
 package org.simple.clinic.benchmark
 
+import com.google.common.truth.Truth.assertThat
 import org.junit.Test
+import org.simple.clinic.TestClinicApp
+import org.simple.clinic.user.User
+import javax.inject.Inject
+import javax.inject.Provider
 
 class CanaryBenchmarkTest : BaseBenchmarkTest() {
 
+  @Inject
+  lateinit var userDaoProvider: Provider<User.RoomDao>
```

Comment:
CMIIW, the reason why we are doing this, is to make sure the user data is inserted in the DB?

Reply:
The reason we are doing this is to add a regression for verifying that the databases are saved and restored across multiple runs of a test.